packages/syntax-floating-components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { StoryObj, Meta } from "@storybook/react";
```

Comment:
@JFChoe The arrow doesn't seem to be in the correct position, is that intended?

![image](https://user-images.githubusercontent.com/127199/234412260-d2e1f4bf-9127-4c8c-9695-adce25a3355f.png)

Reply:
Yes I was having difficulties getting it center at the correct position. I will address this in a separate PR